app/views/feeds/_form_custom_fields.html.erb

```diff
@@ -73,16 +73,16 @@
   <h3><%= t(".headers.ad_zones") %></h3>
   <p><%= t(".help.ad_zones") %></p>
 
-  <%= form.check_box :billboard, value: 0 %>
+  <%= form.check_box :billboard %>
   <%= form.label :billboard, class: "me-2" %>
 
-  <%= form.check_box :house, value: 0 %>
+  <%= form.check_box :house %>
   <%= form.label :house, class: "me-2" %>
 
-  <%= form.check_box :paid, value: 0 %>
+  <%= form.check_box :paid %>
   <%= form.label :paid, class: "me-2" %>
 
-  <%= form.check_box :sonic_id, value: 0 %>
+  <%= form.check_box :sonic_id %>
   <%= form.label :sonic_id, class: "me-2" %>
```

Comment:
Where did this `System (Sonic IDs)` label come from?  Is that in the designs?  I've never heard a sonic_id zone referred to as "system" before... can't decide how confusing I find it yet.  (That term is overridden by Campaign Type in Augury).

Reply:
yes, i think i pulled verbiage from the design.

Reply:
i can change the labels back to parity with Publish and we can bring it up it later.